### ISSUE TITLE
polkit: links to patches are broken

### ISSUE THREAD

**Issue by adnelson** [NixOS/nixpkgs#28084]
## Issue description

Although polkit can be built from the nixpkgs binary cache, building it from scratch produces 404s when trying to fetch the patches.

### Steps to reproduce

```
git clone https://github.com/NixOS/nixpkgs-channels
cd nixpkgs-channels
git checkout nixpkgs-unstable
nix-build polkit --option use-binary-caches false
```

## Technical details

* System: n/a
* Nix version: n/a
* Nixpkgs version: 17.09pre-git
* Sandboxing enabled: n/a

**Comment by lheckemann** [comment]
Are you sure this is polkit and not a dependency of polkit? Could you provide a full build log? I've just built polkit from source successfully using `nix-build '<nixpkgs>' -A polkit --check` after obtaining it from a binary cache. In addition, your command won't work as the file/dir "polkit" does not exist.

**Comment by moxian** [comment]
Encountered the same problem during installation of nixos.
This is about the file at nixpkgs/pkgs/development/libraries/polkit/default.nix - the links in the "patches" list should be changed from `http://pkgs.fedoraproject.org/cgit/rpms/polkit.git/plain/<filename>?id=fa6fd575804de92886c95d3bc2b7eb2abcd13760` to `https://src.fedoraproject.org/rpms/polkit/raw/fa6fd575804de92886c95d3bc2b7eb2abcd13760/f/<filename>`

**Comment by lheckemann** [comment]
Fixed in dfd300c81dfb7292d26cfbce3df9717ea408e410
